modules/player/src/main/java/net/hollowcube/player/event/PlayerBubbleColumnSinkEvent.java

```diff
@@ -0,0 +1,37 @@
+package net.hollowcube.player.event;
+
+import net.minestom.server.entity.Player;
+import net.minestom.server.event.trait.PlayerInstanceEvent;
+import net.minestom.server.instance.block.Block;
+import org.jetbrains.annotations.NotNull;
+
+public class PlayerBubbleColumnSinkEvent implements PlayerInstanceEvent {
```

Comment:
Unsure if an event is strictly necessary right now. It may be useful in the future, but I'm uncertain if it needs to be included here.